src/components/RightSidebar/RightSidebar.vue

```diff
@@ -232,8 +240,17 @@ export default {
 			}
 		},
 	},
-
+	mounted() {
+		this.resize()
+		window.addEventListener('resize', this.resize)
+	},
+	destroyed() {
+		window.removeEventListener('resize', this.resize)
+	},
 	methods: {
+		resize() {
+			this.largeBox = window.innerWidth > 1400
```

Comment:
This evaluates as a boolean here we want a width, right? like `sidebarPlaceholderWidth`